app/frontend/src/components/Examples.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <v-card class="file-input pa-2 my-2">
+    <v-card-title>
+      <p>Example Templates and Sample Data</p>
+    </v-card-title>
+
+    <v-card-text>
+      <v-simple-table>
+        <template v-slot:default>
+          <thead>
+            <tr>
+              <th class="text-left">Description</th>
+              <th v-if="$vuetify.breakpoint.mdAndUp" class="text-left">Type</th>
+              <th class="text-left">Extension</th>
+              <th class="text-left">Template</th>
+              <th class="text-left">Contexts</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr v-for="(item, index) in examples" :key="`item-${index}`">
+              <td>{{ item.description }}</td>
+              <td v-if="$vuetify.breakpoint.mdAndUp">{{ item.type }}</td>
```

Comment:
Can probably change these to the display classes https://vuetifyjs.com/en/styles/display

Reply:
I opted to explicitly leave this in for the table as it has some strange behavior when I tried to use the class based approach instead (alignment issues).

Reply:
Ok, no prob